static/app/views/preprod/buildDetails/main/insights/mainBinaryExportedSymbolsModal.tsx

```diff
@@ -0,0 +1,77 @@
+import {openInsightInfoModal} from 'sentry/actionCreators/modal';
+import {Flex} from 'sentry/components/core/layout';
+import {Text} from 'sentry/components/core/text';
+import {Heading} from 'sentry/components/core/text/heading';
+import {t, tct} from 'sentry/locale';
+import {InlineCode} from 'sentry/views/preprod/buildDetails/main/insights/insightInfoModal';
+
+function getMainBinaryExportedSymbolsContent() {
+  return (
+    <Flex direction="column" gap="2xl">
+      <Flex direction="column" gap="xl">
+        <Flex direction="column" gap="md">
+          <Heading as="h3" size="md">
+            {t('Main Binary Export Metadata')}
+          </Heading>
```

Comment:
IMO delete this heading

in the screenshot it just shows the title twice which seems redundant